pkg/defaults/defaults.go

```diff
@@ -104,8 +104,10 @@ func FromConfig(
 			return nil, nil, fmt.Errorf("could not get Hive client for Hive kube config: %w", err)
 		}
 	}
+	httpClient := retryablehttp.NewClient()
+	httpClient.Logger = nil
 
-	return fromConfig(ctx, config, graphConf, jobSpec, templates, paramFile, promote, client, buildClient, templateClient, podClient, leaseClient, hiveClient, &http.Client{}, requiredTargets, cloneAuthConfig, pullSecret, pushSecret, api.NewDeferredParameters(nil), censor, consoleHost)
+	return fromConfig(ctx, config, graphConf, jobSpec, templates, paramFile, promote, client, buildClient, templateClient, podClient, leaseClient, hiveClient, httpClient.StandardClient(), requiredTargets, cloneAuthConfig, pullSecret, pushSecret, api.NewDeferredParameters(nil), censor, consoleHost)
```

Comment:
here we still pass an http.Client. Is this expected?

Reply:
Yes, it's fine. The client is working the same way as before, it has just modified transport functionality which includes all the benefits of retryable-http.